docs/source/write_first_app.rst

```diff
@@ -43,15 +43,30 @@ After completing this tutorial you should have a basic understanding of how Fabr
 applications and smart contracts work together to manage data on the distributed
 ledger of a blockchain network.
 
+Before you begin
+----------------
+
+In addition to the standard :doc:`prereqs` for Fabric, this tutorial leverages the Hyperledger Fabric SDK for Node.js. See the Node.js SDK `README <https://github.com/hyperledger/fabric-sdk-node#build-and-test>`__ for a up to date list of prerequisites.
+
 Set up the blockchain network
 -----------------------------
 
 If you've already run through :doc:`test_network` tutorial and have a network up
 and running, this tutorial will bring down your running network network before
 bringing up a new one.
 
-If you are using Mac OS and running Mojave, you will need to `install Xcode
-<./tutorial/installxcode.html>`_.
+- If you are using Mac OS, you will need to `install Xcode <./tutorial/installxcode.html>`_.
+- If you are on Windows,  you can install the `windows-build-tools <https://github.com/felixrieseberg/windows-build-tools#readme>`_ with npm which installs all required compilers and tooling by running the following command:
+
+  .. code:: bash
+
+    npm install --global windows-build-tools
+
+- If you are on Linux, you need to install `Python v2.7 <https://www.python.org/download/releases/2.7/>`_, `make <https://www.gnu.org/software/make/>`_, and a C/C++ compiler toolchain such as `GCC <https://gcc.gnu.org/>`_. If you are using Ubuntu, Python is included. You can run the following command to install the other tools:
```

Comment:
The latest version of Ubuntu no longer includes Python by default

Reply:
I removed that sentence.